Marv-Lous/Marv-Lous/Extensions/ComicTableViewCellExtension.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ComicTableViewCellExtension.swift
+//  Marv-Lous
+//
+//  Created by Erik Egers on 2022/03/31.
+//
+
+import Foundation
+
+extension ComicTableViewCell {
```

Comment:
So maybe an enhancement we can make here is how about we extend `String`?  then when you needed to access these functions you could do something like this: 

`link.convertToHttps` 😄 let me know if you have questions.

Reply:
I managed to figure this out.